pkg/controllerutil/pod_utils.go

```diff
@@ -582,3 +584,16 @@ func isContainerFailedAndTimedOut(pod *corev1.Pod, podConditionType corev1.PodCo
 	}
 	return time.Now().After(containerReadyCondition.LastTransitionTime.Add(PodContainerFailedTimeout))
 }
+
+func BuildImagePullSecrets() []corev1.LocalObjectReference {
+	secrets := make([]corev1.LocalObjectReference, 0)
+	secretsVal := viper.GetString(constant.KBImagePullSecrets)
```

Comment:
@leon-inf , should we add `imagePullSecrets` field in ComponentVersion?

Reply:
It is a runtime resource about specific k8s cluster, and is not suitable to define in ComponentVersion.

Reply:
Yeah, then should it be defined in Cluster?

Reply:
It's more reasonable as a global config for KB or k8s cluster.